packages/app-builder-lib/src/node-module-collector/bunNodeModulesCollector.ts

```diff
@@ -0,0 +1,175 @@
+import { log } from "builder-util"
+import * as path from "path"
+import * as resolve from "resolve"
+import { NodeModulesCollector } from "./nodeModulesCollector"
+import { PM } from "./packageManager"
+import { Dependency } from "./types"
+
+interface BunDependency extends Dependency<BunDependency, BunDependency> {
```

Comment:
I believe this should be moved to the types.ts file for node-module-collector since the other types are located there.